### DISCUSSION TITLE
Selecting Minimum Time

### DISCUSSION THREAD

**Discussion by LightningNemesis** [henninghall/react-native-date-picker#338] [Ideas]
Is there a prop via which we can set a minimum time, just like we can set the minimum date? 
When we use the mode='datetime', setting the minimumDate stops the wheel at today (doesn't show anything earlier than today) but the time isn't restricted to the current time only.
Is there any way to implement this?

**Comment by henninghall** [comment]
Correct, the invalid dates are removed from the picker but the hours/minutes are still visible. 

This is not possible to change on iOS because of the limitations of UIDatePicker. On Android this would probably be possible, even though I am not fully aware of all implications. It is an interesting idea, feel free to try to implement it if you like and we can probably include it in the library.

**Reply by nad182** [reply]
I know it's an old issue/discussion, but just thought this might be useful if someone decides to implement this. MUI has a `MobileDateTimePicker` [component](https://github.com/mui/mui-x/blob/master/packages/x-date-pickers/src/MobileDateTimePicker/MobileDateTimePicker.tsx), which exposes props that would solve the OP's problem (I'm sure many people would be happy to see something similar in `react-native-date-picker` as well, at least for Android):
>/**
>   \* Minimal selectable moment of time with binding to date, to set min time in each day use `minTime`.
>\*/
>  minDateTime: PropTypes.any,
>
>/**
>\* If `true`, disable values before the current date for date components, time for time components and both for date time components.
>\* default false
>*/
